geopandas/geoseries.py

```diff
@@ -173,8 +222,15 @@ def copy(self, order='C'):
         copy : GeoSeries
         """
         # FIXME: this will likely be unnecessary in pandas >= 0.13
-        return GeoSeries(self.values.copy(order), index=self.index,
-                      name=self.name).__finalize__(self)
+        return GeoSeries(self._data,
+                         index=self.index, crs=self.crs,
+                         name=self.name)
+
+    def apply(self, func, *args, **kwargs):
+        s = Series(list(self._geometry_array), index=self.index)
```

Comment:
I think you can also do `self.values` (which will call `to_dense` under the hood) to avoid the list intermediate?